designs/PackageVulnerability/DotnetListPackageVulnerable.md

```diff
@@ -165,42 +140,43 @@ The following sources were used:
 
 Package                Requested   Resolved    Latest
 EntityFramework        6.1.2       6.1.2       6.2.0
-NUnit                  2.4.0 (D)   2.6.4       3.8.1  
-My.Sample.Pkg          2.1.3 (V)   4.1.0       4.1.0
+NUnit                  2.4.0       2.6.4 (D)   3.8.1  
+My.Sample.Pkg          2.1.3       4.1.0 (V)   4.1.0
 
+ > To see transitive packages, add the `--include-transitive` option.
 (D): Deprecated package(s). Use 'dotnet list package --deprecated' for more info.
 (V): Vulnerable package(s). Use 'dotnet list package --vulnerable' for more info.
 ```
+Notes:
+* *Important! Installed packages that are vulnerable or deprecated but not outdated will not be part of this command's output.*
```

Comment:
Can we clarify this in the `dotnet list package --outdated` output? In its current state one could easily mistakenly believe this included all deprecated/vulnerable packages.

Reply:
If we were to do this, we would need to make equivalent changes to `--deprecated` and `--vulnerable` reports. I think this is a good addition, but I'm also wondering just how "chatty" we want to make our output. And perhaps the answer is "quite chatty". :) If these are helpful tips, perhaps they should be included.

Reply:
Definitely agreed. Maybe we should participate in an UX review on the messaging? @chgill-MSFT knows a "word smith" (I don't know his official title, but he specializes on crafting clear yet concise messaging).

My suggestions:
1. Consider changing `Use 'dotnet list package --deprecated' for more information.` to  `Use 'dotnet list package --deprecated' to list all deprecated packages.`.
2. Reconsider this phrasing: `3 packages need your attention - 2 outdated, 1 deprecated, 1 vulnerable`. The messaging indicates the counts are exact, which is not true for deprecated and vulnerable in this example. Maybe use the phrasing `3 outdated packages need your attention` instead.

Reply:
Per @JonDouglas's comment above, we'll remove footers for now and let customers use `--help` for the full help experience.